styles.css

```diff
@@ -0,0 +1,971 @@
+	border-right: 1px solid var(--color-grey-dark);
+	border-top: 1px solid var(--color-grey-dark);
+	border-radius: 0.5rem;
+	padding: 0;
+	overflow: hidden;
+}
+.list-group-item + .list-group-item{
+	border-top: 1px solid var(--color-grey-dark);
+}
+.list-group-item{
+	margin: 0;
+	padding: 0.5rem;
+	min-width: 10rem;
+}
+.list-group .active{
+	background-color: var(--color-primary);
+	color: white;
+}
+/* Badges */
+
+.badge {
+	border: 1px solid transparent;
+	border-radius: 0.25rem;
+	color: white;
+	padding: 0.25rem 0.5rem;
+	font-size: 0.75rem;
+	font-weight: 700;
+}
+
+.ic-bg {
+	font-size: 200%;
+}
+
+.icon-badge {
+	/* height: 2rem;
+    width: 2rem; */
+	border-radius: 1rem;
+	font-size: 1.5rem;
+	/* display: flex;
+    align-items: center;
+    justify-content: center; */
+	background-color: var(--color-primary-bg);
+}
+
+.bg-of-icon {
+	position: relative;
+	top: -1rem;
+	right: -1rem;
+	z-index: 1;
+}
+
+.rounded-pill {
+	border-radius: 1rem;
+	/* opacity: 70%; */
+}
+
+.bg-primary {
+	background-color: var(--color-primary);
+}
+
+.bg-secondary {
+	background-color: var(--color-secondary);
+}
+
+.bg-success {
+	background-color: var(--color-success);
+}
+
+.bg-danger {
+	background-color: var(--color-danger);
+}
+
+.bg-warning {
+	background-color: var(--color-warning);
+}
+
+.bg-info {
+	background-color: var(--color-secondary);
+}
+
+.avatar-badge-container {
+	position: relative;
+}
+
+.avatar-badge-top {
+	min-width: 1rem;
+	height: 1rem;
+	padding: 0.15rem;
+	border-radius: 50%;
+	border: 1px solid white;
+	background: var(--color-primary);
+	color: var(--color-grey-light);
+	position: absolute;
+	top: 0%;
+	right: 0%;
+	font-size: 0.65rem;
+	font-weight: bold;
+}
+
+.avatar-badge-bottom {
+	min-width: 1rem;
+	height: 1rem;
+	padding: 0.15rem;
+	border-radius: 50%;
+	border: 1px solid white;
+	background: var(--color-primary);
+	color: var(--color-grey-light);
+	position: absolute;
+	bottom: 2%;
+	right: 2%;
+	font-size: 0.65rem;
+	font-weight: bold;
+}
+
+.avatar-badge-container .fa {
+	color: var(--text-color);
+	font-size: 200%;
+	-webkit-text-stroke-width: 1px;
+	-webkit-text-stroke-color: var(--darkgrey);
+}
+
+.icon-badge-top {
+	min-width: 1rem;
+	height: 1rem;
+	padding: 0.15rem;
+	border-radius: 0.5rem;
+	background: var(--color-primary);
+	color: var(--color-grey-light);
+	border: 1px solid white;
+	position: absolute;
+	top: 10%;
+	right: 10%;
+	font-size: 0.65rem;
+	font-weight: bold;
+}
+
+/* card */
+.cards-container {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	display: border-box;
+}
+
+.card-product {
+	position: relative;
+	transition: all 200ms ease-in-out;
+	cursor: pointer;
+	max-width: 200px;
+	border: 1px solid var(--color-grey-bg);
+	display: border-box;
+	overflow: hidden;
+}
+
+.card-img-container {
+	width: 100%;
+	overflow: hidden;
+}
+
+.responsive-img {
+	width: 100%;
+	height: 100%;
+}
+
+.card-description {
+	padding: 0.15rem 0.5rem;
+	display: block;
+	flex-direction: column;
```

Comment:
Remove `flex-direction` as `display: block` is given so it won't work, if you want to make it work replace `block` with `flex`